lib/rules/use-ember-get-and-set.js

```diff
@@ -13,33 +17,80 @@ module.exports = {
       category: 'General',
       recommended: true
     },
-    fixable: null, // or "code" or "whitespace"
+    fixable: 'code',
   },
 
   create(context) {
+    const sourceCode = context.getSourceCode();
+    const filename = context.getFilename();
     const message = 'Use get/set';
 
-    const report = function (node) {
-      context.report(node, message);
-    };
+    function isFileInDirectory(dirname) {
+      const pathParts = filename.split(path.sep);
+      return pathParts.indexOf(dirname) > -1;
+    }
+
+    function report(node) {
+      context.report({
+        node: node.callee.property,
+        message,
+        fix(fixer) {
+          // this.set('foo', 3);
+          // └┬─┘ └┬┘ └───┬──┘
+          //  │    │      └─────── args
+          //  │    └────────────── method
+          //  └─────────────────── subject
+
+          const subject = sourceCode.getText(node.callee.object);
+          const method = sourceCode.getText(node.callee.property);
+          const args = node.arguments.map(a => sourceCode.getText(a)).join(', ');
+
+          const fixedSource = `${method}(${subject}, ${args})`;
 
-    const avoidedProperties = [
+          return fixer.replaceText(node, fixedSource);
+        }
+      });
+    }
+
+    const avoidedMethods = [
       'get',
       'set',
       'getProperties',
       'setProperties',
       'getWithDefault',
     ];
 
+    const testMethodsToSkip = [
+      'get',
+      'set',
+    ];
+
+    const directoriesToSkipCompletely = [
```

Comment:
Makes sense to me. I was considering this, but I didn't have a great way to specify the option. I was thinking either:
- Only allow them to specify a single part of the path as we do here.
- Substring match. Not great and could lead to false positives. 
- Substring match but make sure that we're respecting path boundaries (e.g. `app/nottests` would not match to `tests`.
- Regex match. Could be a little heavy handed.
- Glob matching. Would probably require requiring in another dependency.

Alternatively, the users could include a `.eslintrc` file in the subdirectory they want to exclude.
```javascript
// some/subdirectory/to/exclude/.eslintrc
{
  "ember/use-ember-get-and-set": "off"
}
```